docs/source/docs.rst

```diff
@@ -6,26 +6,26 @@ Dask
 
 Dask is composed of two parts:
 
-1.  **Dynamic task scheduling** optimized for computation.  This is similar to
+1.  **Dynamic task scheduling** optimized for computation. This is similar to
     *Airflow, Luigi, Celery, or Make*, but optimized for interactive
     computational workloads.
-2.  **"Big Data" collections** like parallel arrays, dataframes, and lists that
-    extend common interfaces like *NumPy, Pandas, or Python iterators* to
```

Comment:
Technically these are [Oxford commas]( https://en.wikipedia.org/wiki/Serial_comma ), which are legal (albeit obscure) grammatical devices in English.

Reply:
You are correct. Oxford commas are grammatically correct (and useful) and its use depends on the style guide being followed.

The main issue I've seen throughout the documentation is consistency (with punctuation and naming conventions). Sometimes I see the use of serial commas in lists, and sometimes I don't. The reason for me removing the use of serial commas in the text is that, as a non native English speaker, I've been taught (British) English and all publications I've read or written did not make use of the serial comma.

It would be good to know which one the devs would like to be used in order to have a consistent rule throughout the documentation that can be followed.

Reply:
I probably default to using the Oxford/serial comma, but would be fine either way.  I would be happy to let you two decide what we do here.  

The rest of this PR looks great to me.  Thank you as always @farrajota

Reply:
Since the documentation is written in en-us, it makes sense to follow the style guide used by the majority of US publishers / institutions which mandate the use of the serial comma.

But if the goal is to make it easier for non native English speakers then maybe it would be of interest to evaluate the impact of using the serial comma as mandatory. For example, many European languages don't use serial commas, and I expect many other languages around the world also don't use it either. That does not mean that people can't understand it... it just feels weird.

Regardless of what is chosen, the thing that should matter most here is consistency. That is why I suggest further discussion about this topic in order to come to a consensus.

What do you guys think?

Reply:
I wonder if @electronwill has thoughts here

Reply:
> That is why I suggest further discussion about this topic in order to come to a consensus.

@farrajota it looks like there hasn't been much interest on this topic.  You could also try the gitter chat channel at https://gitter.im/dask/dev .  In general though I suspect that most people don't have a strong opinion.

Reply:
Later on I'll revert the changes related to serial commas and start using them as a general rule from now on. 

In the future, if it is decided to go against the use of serial commas then it will be only a matter of updating the docs here and there.